client/src/www/app/outline_server_repository/server.cordova.ts

```diff
@@ -23,8 +23,12 @@ export class CordovaTunnel implements PlatformTunnel {
     if (!config) {
       throw new errors.IllegalServerConfiguration();
     }
-    return pluginExecWithErrorCode<void>('start', this.id, name, config);
-  }
+    if (cordova.platformId === 'android') {
```

Comment:
Please revert this file and pull master. The wiring is already implemented for all platforms in https://github.com/Jigsaw-Code/outline-apps/pull/2105.